src/runtime/virtcontainers/kata_agent.go

```diff
@@ -364,6 +365,11 @@ func (k *kataAgent) init(ctx context.Context, sandbox *Sandbox, config KataAgent
 	k.kmodules = config.KernelModules
 	k.dialTimout = config.DialTimeout
 
+	imageRequestTimeout = time.Duration(sandbox.config.ImageRequestTimeout) * time.Second
+	k.Logger().WithFields(logrus.Fields{
+		"imageRequestTimeout": fmt.Sprintf("%+v", imageRequestTimeout),
+	}).Warn("The imageRequestTimeout has been set ", imageRequestTimeout, " seconds.")
```

Comment:
Probably more an `Info` than a `Warn`. Unless there is some negative effect?

Reply:
Ok. I'll update it.